cmd/appDeploy.go

```diff
@@ -79,6 +80,7 @@ func init() {
 	appCmdDeploy.Flags().StringVarP(&deployApp.userName, "username", "u", "", "Username of private container registry")
 	appCmdDeploy.Flags().StringVarP(&deployApp.password, "password", "P", "", "Password of private container registry")
 	appCmdDeploy.Flags().StringArrayVarP(&deployApp.env, "env", "e", nil, "Environment variable to set, as key=value pair")
+	appCmdDeploy.Flags().StringVarP(&deployApp.envFilePath, "envPath", "f", "", "File path for Enviroment variables, as line separated key=value pair")
```

Comment:
NiT: "Path to the environment variables file."
Also lets include format of values something like
"Values in the .env file should be formatted as a line separated Key=Value pair"